libcxx/utils/generate_iwyu_mapping.py

```diff
@@ -22,6 +22,8 @@ def IWYU_mapping(header: str) -> typing.Optional[typing.List[str]]:
         return ["bits"]
     elif header in ("__bit_reference", "__fwd/bit_reference.h"):
         return ["bitset", "vector"]
+    elif re.match("__configuration/.+", header):
```

Comment:
I was about to ask for a comment, but to me this makes the intention clear.
```suggestion
    elif re.match("__configuration/.+", header) or header == "__config":
```